src/components/gcds-banner/test/gcds-banner.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { newE2EPage } from '@stencil/core/testing';
+const { AxePuppeteer } = require('@axe-core/puppeteer');
+
+describe('gcds-banner', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<gcds-banner></gcds-banner>');
+
+    const element = await page.find('gcds-banner');
+    expect(element).toHaveClass('hydrated');
+  });
```

Comment:
Can we add a few a11y tests here? Just colour contrast tests on the different roles.

Reply:
Added colour contrast tests for all roles.